### PR TITLE
Create Timeline-component.html

### PR DESCRIPTION
**Pull request by Kumar-Ankit56**
successfully resolve the  #685.

This is the final output made in tailwind

Screenshot (55)

### CODE REVIEW

File: Others/Timeline-component.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (Rajdip019) [inline Others/Timeline-component.html:2]: Remove the HTML boiler template.

**Comment by Kumar-Ankit56** [comment]
@Rajdip019  check ones

File: Others/Timeline-component.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
+  <head>
```
Comment (Rajdip019) [inline Others/Timeline-component.html:1]: Remove this `head` as well.

Comment (Kumar-Ankit56) [reply Others/Timeline-component.html]: Done everything.

**Comment by Rajdip019** [comment]
Made some small change else everything was fine. Look into the changes and great work by the way.

Congrats 🎉, your PR got merged. To be eligible for the swags from ClueLess. Follow these steps.

1. Star this repository. ⭐
2. SignIn / SignUp in [[ClueLess Website](https://www.clueless.tech/)](https://www.clueless.tech/)
3. Go to the profile section from the top right-hand corner.



Comment your profile URL under this comment tagging the person who merged your PR.

That’s it. Follow the leaderboard!

And don’t forget to share your achievement with your friends.

**Comment by Kumar-Ankit56** [comment]
@Rajdip019  from where I can access leaderboards as I have signed up.

**Pull request merged**